server/rooms.ts

```diff
@@ -373,7 +375,7 @@ export abstract class BasicRoom {
 		}
 		this.runMuteTimer();
 
-		user.updateIdentity(this.roomid);
+		user.updateIdentity();
```

Comment:
?

Reply:
You need to update it globally now, since you might also be getting muted / unmuted in a subroom.

Reply:
ohhhh